src/App.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
```

Comment:
state should have not a text but a pressed key
```suggestion
    pressedKey: '',
```